odoo/tools/pdf/__init__.py

```diff
@@ -437,10 +437,8 @@ def convert_to_pdfa(self):
         # " The aforementioned EOL marker shall be immediately followed by a % (25h) character followed by at least four
         # bytes, each of whose encoded byte values shall have a decimal value greater than 127 "
         self._header = b"%PDF-1.7"
-        if SUBMOD != '._pypdf2_2':
-            self._header += b"\n"
-        if SUBMOD == '._pypdf2_1':
-            self._header += b"%\xDE\xAD\xBE\xEF"
+        if SUBMOD in ('._pypdf', '._pypdf2_1'):
+            self._header += b"\n%\xc3\xa9\xc3\xa9\xc3\xa9\xc3\xa9"
```

Comment:
Can you comment that part just to have a vague idea from where those characters come from plz